packages/website-frontend/test/unit/pages/repositories.page.spec.ts

```diff
@@ -3,16 +3,24 @@ import '@stryker-mutator/stryker-elements';
 
 import { CustomElementFixture } from '../../helpers/custom-element-fixture';
 import { RepositoriesPage } from '../../../src/pages/repositories.page';
-import { locationService } from '../../../src/services/location.service';
+import { authService } from '../../../src/services/auth.service';
+import { historyService } from '../../../src/services/history.service';
 import { organizationsService } from '../../../src/services/organizations.service';
 import { repositoriesService } from '../../../src/services/repositories.service';
 import { userService } from '../../../src/services/user.service';
+import { Mock } from 'vitest';
 
 describe(RepositoriesPage.name, () => {
   let sut: CustomElementFixture<RepositoriesPage>;
+  let pushStateMock: Mock;
 
   beforeEach(() => {
-    locationService.getLocation = vi.fn(() => ({ pathname: '/repos/user' }) as Location);
+    vi.spyOn(authService, 'currentUser', 'get').mockReturnValue({
+      avatarUrl: 'foo',
+      name: 'mockUser',
+    });
+    pushStateMock = vi.fn();
+    historyService.getHistory = vi.fn(() => ({ pushState: pushStateMock }) as unknown as History);
```

Comment:
Instead of mocking, can't you test the real URI of the browser?

Reply:
We might be able to, if playwright doesn’t keep overwriting it.

Reply:
I was able to, pushing history apparently does not affect playwright 🙂